vmm/src/vm.rs

```diff
@@ -1361,6 +1361,11 @@ impl Vm {
             .read_raw(&mut out)
             .map_err(Error::Console)?;
 
+        // Replace "\n" with "\r" to deal with Windows SAC (#1170)
+        if count == 1 && out[0] == 0x0a {
+            out[0] = 0x0d;
```

Comment:
I thought that a `\n` on Unix was translating into `\r\n` on Windows. I'm just asking since there might be some cases where this still doesn't work.

Reply:
I honestly don't know why this works since I can't see the code that handles this in Windows. But I know that it does and that there is no obvious side effect with Linux and getty.